src/main/resources/db/migration/V1_88__scheduled_job_performance_stats.sql

```diff
@@ -0,0 +1,665 @@
+-- Amended : 27/02/2023 JDB-5346 Defect fix for jurors whose original pool has been deleted
+-- Amended : 02/03/2023 JDB-5349 The dashboard should ignore the pool reasignment history event 'PREA'
+-- Amended : 02/05/2023 JDB-5374 The dashboard should ignore the summons reprinted 'RSUP' events (LH)
+-- Amended : 13/03/2024 JM-6544 Refactor for Juror Modernisation changes to the data model
+
+-- entry point
+declare
+
+
+begin
+
+	call juror_dashboard.auto_processed();
+	call juror_dashboard.response_times_and_not_responded(response_times_months);
+	call juror_dashboard.unprocessed_responses();
+	call juror_dashboard.welsh_online_responses(welsh_online_months);
+	call juror_dashboard.thirdparty_online(third_party_months);
+	call juror_dashboard.excusals(excusals_months);
+	call juror_dashboard.deferrals(deferrals_months);
+
+end;
+
+$$;
+
+
+create or replace procedure juror_dashboard.auto_processed()
+language plpgsql as
+
+$$
+/**
+ * Populates the stats_auto_processed table, runs 7 days a week and uses
+ * max processed date to figure out the last date processed.
+ *
+ * No insertion of rows for days that have no AUTO processed responses
+ *
+ * Digital responses only.
+ *
+ * It will automatically catch-up if it hasn't been run for one or more days.
+ *
+ * To refresh existing rows, those rows would first need to be deleted from stats_auto_processed.
+ */
+declare
+
+	v_text_var1 text;
+   	v_text_var2 text;
+   	v_text_var3 text;
+	l_Job_Type	varchar(50);
+
+begin
+
+	l_Job_Type := 'refresh_stats_data.auto_processed';
+
+	insert into juror_dashboard.stats_auto_processed (
+
+		select		r.staff_assignment_date::date processed_date,
+					count(1) as "count"
+		from		juror_mod.juror_response r
+		where		r.staff_login = 'AUTO'
+					and r.staff_assignment_date::date > (
+						select	coalesce(max(processed_date), '01-JAN-1990')
+						from	juror_dashboard.stats_auto_processed)
+					and r.staff_assignment_date::date < current_date
+					and r.reply_type = 'Digital'
+		group by	r.staff_assignment_date::date
+	);
+
+exception
+
+	when others then
+	        get stacked diagnostics v_text_var1 = message_text,
+	                                v_text_var2 = pg_exception_detail,
+	                                v_text_var3 = pg_exception_hint;
+
+	raise notice '%', 'auto process failed - error:->' || v_text_var1 || '|' || v_text_var2 || '|' || v_text_var3;
+
+	rollback;
+
+end;
+
+$$;
+
+
+
+create or replace procedure juror_dashboard.response_times_and_not_responded(no_of_months int)
+language plpgsql as
+
+$$
+ /* Populates juror_dashboard.stats_not_responded and juror_dashboard.stats_response_times
+  * used by the Juror Digital performance dashboard
+  *
+  * Populating both tables in a single transacton so that they are always consistant with each other
+  *
+  * Replaces the latest n months of summons counts as specified by the no_of_months input parameter
+  *
+  * It is recommended to use use no_of_months = 6
```

Comment:
Could we update the comment to remove the duplicated use please?

Reply:
Removed